packages/api/src/mailing/entities/mailing-entity.factory.ts

```diff
@@ -0,0 +1,80 @@
+import { Block, BlockDataInterface, RootBlock } from "@comet/blocks-api";
+import { DocumentInterface, RootBlockDataScalar, RootBlockType } from "@comet/cms-api";
+import { Embedded, Entity, OptionalProps, PrimaryKey, Property } from "@mikro-orm/core";
+import { Type } from "@nestjs/common";
+import { Field, ID, Int, ObjectType } from "@nestjs/graphql";
+import { MailingScopeInterface } from "src/types";
+import { v4 } from "uuid";
+
+import { SendingState } from "../sending-state.enum";
+
+export interface MailingInterface {
+    [OptionalProps]?: "createdAt" | "updatedAt" | "sendingState";
+    id: string;
+    createdAt: Date;
+    scheduledAt?: Date;
+    title: string;
+    subject: string;
+    brevoId?: number;
+    updatedAt: Date;
+    content: BlockDataInterface;
+    scope: MailingScopeInterface;
+}
+
+export class MailingEntityFactory {
```

Comment:
I think Mailing is too generic.
I would rather call it CampaignEntityFactory, NewsletterCampaignEntityFactory or something like that

Reply:
I renamed Mailing to Campaign in [cb4c4f3](https://github.com/vivid-planet/comet-brevo-module/pull/8/commits/cb4c4f39ca75b1000e8a32b0e72570418a8da875)

Reply:
As discussed I renamed it to EmailCampaign in [a05f14a](https://github.com/vivid-planet/comet-brevo-module/pull/8/commits/a05f14a8a3ab5252ce00e9aa7aab6c8a8642b9f2)